enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/DelegatingActivationCodeHandler.java

```diff
@@ -58,7 +58,7 @@ public interface DelegatingActivationCodeHandler {
      * @return List of new activation flags for the destination activation.
      * @throws ActivationCodeException Thrown in case activation flag processing fails.
      */
-    List<String> addActivationFlags(String sourceActivationId, List<String> sourceActivationFlags, String userId, String applicationId, Long sourceAppId, List<String> sourceApplicationRoles, Long destinationAppId, String destinationActivationId, String activationCode, String activationCodeSignature) throws ActivationCodeException;
+    List<String> addActivationFlags(String sourceActivationId, List<String> sourceActivationFlags, String userId, String applicationId, String sourceAppId, List<String> sourceApplicationRoles, String destinationAppId, String destinationActivationId, String activationCode, String activationCodeSignature) throws ActivationCodeException;
```

Comment:
Ok for this PR, but in general, too many arguments of the same type. Very error prone for developers. Could be nice to refactor to a parameter object.